docs/setup-robusta/alertsui.rst

```diff
@@ -38,36 +33,84 @@ Disable Default Prometheus Alert Rules
 
 Since Robusta creates new PrometheusRule custom resources, you must disable the default Kube Prometheus Stack alerts to avoid duplication of alerts.
 
-Add the following config to your ``generated_values.yaml``.
+Choose the appropriate instructions below, based on whether you use the Prometheus bundled with Robusta or your own Prometheus.
+
+.. tab-set::
 
-.. code-block:: yaml
+.. tab-set::
+   .. tab-item:: Robusta’s Prometheus
 
-    enabledManagedConfiguration: true # (1)
-    kube-prometheus-stack:
-        defaultRules:
+      To start syncing alerts to your cluster and to avoid duplication, add the following snippet to Robusta’s Helm values file named ``generated_values.yaml``:
+
+      .. code-block:: yaml
+
+        enabledManagedConfiguration: true # Enable managed alerts (See 1.)
+        kube-prometheus-stack: # those rules are now managed by Robusta
+          defaultRules:
+            rules:
+              alertmanager: false
+              etcd: false
+              configReloaders: false
+              general: false
+              kubeApiserverSlos: false
+              kubeControllerManager: false
+              kubeProxy: false
+              kubernetesApps: false
+              kubernetesResources: false
+              kubernetesStorage: false
+              kubernetesSystem: false
+              kubeSchedulerAlerting: false
+              kubeStateMetrics: false
+              network: false
+              nodeExporterAlerting: false
+              prometheus: false
+              prometheusOperator: false
+
+      Then perform a :ref:`Helm Upgrade <Simple Upgrade>`.
+
+   .. tab-item:: External Prometheus
+
+      First, ensure you have the Prometheus operator installed by running the following command:
+
+      .. code-block:: bash
+
+         kubectl get crd | grep prometheus
+
+      To make sure Prometheus picks up Robusta's rule files and avoid duplication, add the following to the kube-prometheus-stack configuration:
```

Comment:
Please change **kube-prometheus-stack** to **Kube Prometheus Stack** since we use it everywhere else on this page.